agent/util-scripts/unittests

```diff
@@ -315,12 +315,8 @@ declare -A tools=(
     [test-30]="pbench-collect-sysinfo"
     [test-31]="pbench-copy-results"
     [test-32]="pbench-copy-results"
-    [test-33]="pbench-move-results"
```

Comment:
Why do we need to remove these tests?

Reply:
This test has moved to python so I removed the duplicate unit test.